azerothcore/azerothcore-wotlk
# fix(DB/Creature): Lich King Audio and Text during Chapter 3 of DK Questline

## Changes Proposed:
-  Applies Path of Frost aura to Lich King during Phase 3 of the Ebon Hold questlines.  Spell ID is already applied to the phase 1 counterpart.
-  Add audio, create text group and apply SAI to Lich King during Phase 3 of Ebon Hold.  Will cycle through 17 voice lines over the course of 17 minutes (60 sec interval between voices).  The voicefiles are called A_EH_LichKing_Chapter3Shout01-17 to confirm that these are in the right place.

## SOURCE:
https://youtu.be/b863yUtcCzM?t=4920
https://youtu.be/kqqxXo9FEbI?t=3628

https://www.wowhead.com/npc=29110/the-lich-king quotes also list Battle of Light's Hope for some reason

## Tests Performed:
I applied the SQL and sat and wait while the npc went through the voice files.  There are 17 in total.

[With Aura](https://i.imgur.com/X3fl0Hj.jpg)

[Voice Files](https://i.imgur.com/e4IJgYM.jpg)

## How to Test the Changes:
.go c 130896
make sure aura is under his feet
make sure he cycles through text and voice lines
